Base/src/test/java/io/deephaven/base/FileUtilsTest.java

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.base;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+
+import junit.framework.TestCase;
+import org.junit.Assert;
+import org.junit.rules.TemporaryFolder;
+
+public class FileUtilsTest extends TestCase {
+
+    public void testConvertToFileURI() throws IOException {
+        final TemporaryFolder folder = new TemporaryFolder();
+        folder.create();
```

Comment:
Seems odd to me that we actually want to create a temporary folder just so we can test URI stuff. There may be something valid about wanting this to work on Windows as well as Unix, but you could just do it based on the current directory; `Path.of("").toAbsolutePath().toString()`.